modules/server/src/main/scala/scaladex/server/service/AdminService.scala

```diff
@@ -74,6 +74,43 @@ class AdminService(
 
   def allTaskStatuses: Seq[Task.Status] = tasks.map(_.status)
 
+  def addProjectNoArtifact(reference: Project.Reference, user: UserState
+                          ): Unit = {
+
+    def buildSettings(): Settings = Settings(
+      false, None, false, None, List.empty, false,false, Set.empty, Set.empty, None, None)
+
+    def buildProject(info: GithubInfo): Project = new Project(
+      organization = Organization(""), // FIXME
+      repository = Repository(""), // FIXME
+      creationDate = None, // FIXME
+      GithubStatus.Ok(java.time.Instant.now),
+      githubInfo = Some(info),
+      settings = buildSettings()
+    )
+
+    val input = Seq("Organization" -> reference.organization.value, "Repository" -> reference.repository.value)
+
+    val task = TaskRunner.run(Task.missingProjectNoArtifact, user.info.login, input) { () =>
+      githubClientOpt.fold(throw new Exception("Failed because 1 ??")) { githubClient =>
+        githubClient.getProjectInfo(reference).flatMap {
+          case GithubResponse.Ok((_, info)) =>
```

Comment:
You should also handle other possible `GithubResponse` cases, possibly with an error message.